joblib/func_inspect.py

```diff
@@ -169,7 +170,11 @@ def getfullargspec(func):
     try:
         return inspect.getfullargspec(func)
     except AttributeError:
-        arg_spec = inspect.getargspec(func)
+        if 3 > sys.version_info[0]:
+            arg_spec = inspect.getargspec(func)
```

Comment:
@lesteve I think I see your point.  However, if I'm understanding then [inspect.getfullargspec is deprecated at 3.5 ](https://docs.python.org/3/library/inspect.html)so should work but with the warning.  But I have not been able to produce any of the deprecation warnings seen in scikit-learn.  I'm pretty much a novice here so I'm not certain how make a test for this slice of joblib.